pages/models.py

```diff
@@ -117,12 +117,12 @@ class Meta:
     )
 
     menu_title = models.CharField(
-        max_length=255, blank=True,
+        max_length=1000, blank=True,
         help_text=_('This field will be shown in the breadcrumbs, menu items and etc.'),
         verbose_name=_('menu title'),
     )
 
-    h1 = models.TextField(blank=True)
+    h1 = models.CharField(max_length=1000, blank=True)
```

Comment:
Why is not `TextField`?

Reply:
@artemiy312 , because text field looks like field in admin panel. But charField - like one text string

Reply:
@duker33, you should [change a widget](https://docs.djangoproject.com/en/2.1/ref/contrib/admin/#django.contrib.admin.ModelAdmin.formfield_overrides) for the field instead of its type. From [TextArea](https://docs.djangoproject.com/en/2.1/ref/forms/widgets/#textarea) to [TextInput](https://docs.djangoproject.com/en/2.1/ref/forms/widgets/#django.forms.TextInput)

Reply:
@duker33, you should change a widget for the field instead of its type. From TextArea to TextInput, otherwise you have to create a migration for it

Reply:
@artemiy312 , y, i'll create migration.
name, h1, slug, etc are not texts but only long labels by it's semantic